### ISSUE TITLE
Opt-in should be a configuration option

### ISSUE THREAD

**Issue by andrewcmyers** [andrewcmyers/civs#45]
Currently it is always on.

**Comment by danielcompton** [comment]
We ([Clojurists Together](https://www.clojuriststogether.org)) are using CIVS to distribute ballots to our 200+ members. Making the ballots opt-in is quite tricky as we'll need to keep track of which users have told us that they have opted in so we can send their ballot.

I'd be supportive of a manual verification step for poll senders to change this to opt-out. Perhaps this could be coupled with paying a fee to prevent this being an attractive spam tool?

**Comment by andrewcmyers** [comment]
When you send ballots out, you'll get a list (and a CSV file, if you like) of all the users who haven't opted in, so you can then communicate with them out of band. You can always resend the ballot link later.

**Comment by danielcompton** [comment]
Yep, the CSV is definitely very helpful but it's still a fairly laborious process to handle. This is our problem, not yours though! Thanks for CIVS!
